microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -409,6 +409,19 @@ The shopping service is turned on by default for all users.  To change the shopp
 
 InPrivate browsing uses the shopping setting of the profile that launched the InPrivate session.
 
+## Travel
+
+When you do online activities related to travelling, Microsoft Edge helps you find recommendations for travel.  To help you find recommendations while planning your travel online, Microsoft Edge downloads a list of travel domains to the client from the Microsoft Travel service.
+
+When you visit a website, Microsoft Edge locally determines if the website you're on is a travel domain.  If the website is identified as a Travel-related webpage, Microsoft Edge sends the domain, flight dates, From and To locations, and passenger count, along with information about Microsoft Edge and cookies (if cookies are allowed) to the service.  This data does not include any personally identifiable information, and is sent over HTTPS.
+
+The Microsoft Edge Travel feature requires sharing cookie information with Bing.com.  For example, cookies may be used for debugging, fraud detection, and analytics.  When you visit Bing.com in your browser and update any settings on Bing pages, Bing.com creates a cookie in your browser and stores information in the cookie.  This cookie is shared across Bing.com pages, and Microsoft Edge sends this cookie to the Microsoft Travel service to keep your experience consistent.
+
+<!-- todo: revise/delete/update.  'travel' is not found in Settings in stable or in Canary 96 -->
```

Comment:
@chgangwa - the word 'travel' doesn't appear in Settings.  Is this a future feature?  Delete this paragraph?